charts/keycloak/templates/_helpers.tpl

```diff
@@ -166,11 +166,16 @@ Create environment variables for database configuration.
   value: {{ .Values.keycloak.persistence.dbName | quote }}
 - name: DB_USER
   value: {{ .Values.keycloak.persistence.dbUser | quote }}
+{{- if .Values.keycloak.persistence.existingSecret}}
+- name: DB_PASSWORD_FILE
+  value: /secrets/persistence
+{{- else }}
```

Comment:
I'd suggest we completely move to using password files only. This is already the case for the Keycloak password.

Reply:
I don't know if anybody wants to still use only the env variable, the docker container allows people to do so... I tried to be backward compatible

Reply:
I don't think we will have a compatibilty issue here. I think an upgrade should not be an issue.

Reply:
your call... I don't have visibility on all possible hacks done by people on their systems out there